apps/website/database/migrations/1743597023337_alter_events_table.ts

```diff
@@ -0,0 +1,18 @@
+import { BaseSchema } from "@adonisjs/lucid/schema";
+
+export default class extends BaseSchema {
+  protected tableName = "events";
+
+  async up() {
+    this.schema.alterTable(this.tableName, (table) => {
+      table.text("extra_info");
+      table.text("description").nullable().alter();
```

Comment:
Not sure how lucid handles this but shouldn't you also reset `description` during the `down` method?

Reply:
yes